proposals/nnnn-deliver-build-configuration.md

```diff
@@ -0,0 +1,77 @@
+# Deliver build configuration
+
+* Proposal: [SDL-NNNN](nnnn-deliver-build-configuration.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+Add text file with information about SDL build configuration as part of SDL delivery. 
+
+## Motivation
+
+After installation of SDL (`make install`) there are couple of files besides SDL binary:
+
+- audio.8bit.wav         
+- libPolicy.so        
+- mycert.pem  
+- sample_policy_manager.py  
+- smartDeviceLink.ini
+- hmi_capabilities.json  
+- libUtils.a          
+- mykey.pem   
+- sdl_preloaded_pt.json     
+- start.sh
+- libappenders.so        
+- log4cxx.properties  
+- plugins     
+- smartDeviceLinkCore       
+
+
+Only these files (except 3rd party libraries) are required for SDL testing.
+But there is no information about flags that were used for SDL compiling. 
+And there is no way to find out it only from delivered files.
```

Comment:
Can you please re-word?  I believe this should say "And there is no way to find out this information from delivered files."

Reply:
@theresalech fixed